diff -up wxPython-src-2.8.7.1/wxPython/wxPython/_misc.py~ wxPython-src-2.8.7.1/wxPython/wxPython/_misc.py
--- wxPython-src-2.8.7.1/wxPython/wxPython/_misc.py~	2008-06-12 23:45:44.000000000 +0200
+++ wxPython-src-2.8.7.1/wxPython/wxPython/_misc.py	2008-06-12 23:45:44.000000000 +0200
@@ -453,7 +453,7 @@ wxDateTime_ConvertYearToBC = wx._misc.Da
 wxDateTime_GetCurrentMonth = wx._misc.DateTime_GetCurrentMonth
 wxDateTime_IsLeapYear = wx._misc.DateTime_IsLeapYear
 wxDateTime_GetCentury = wx._misc.DateTime_GetCentury
-wxDateTime_GetNumberOfDaysinYear = wx._misc.DateTime_GetNumberOfDaysinYear
+wxDateTime_GetNumberOfDaysInYear = wx._misc.DateTime_GetNumberOfDaysInYear
 wxDateTime_GetNumberOfDaysInMonth = wx._misc.DateTime_GetNumberOfDaysInMonth
 wxDateTime_GetMonthName = wx._misc.DateTime_GetMonthName
 wxDateTime_GetWeekDayName = wx._misc.DateTime_GetWeekDayName
