diff -up wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_pages.py.aui wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_pages.py
--- wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_pages.py.aui	2010-05-27 15:38:42.000000000 +0200
+++ wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_pages.py	2010-05-27 15:38:50.000000000 +0200
@@ -36,7 +36,7 @@ import ed_txt
 import ed_mdlg
 import ebmlib
 import eclib
-from extern import aui
+from wx.lib.agw import aui
 
 #--------------------------------------------------------------------------#
 # Globals
diff -up wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_shelf.py.aui wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_shelf.py
--- wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_shelf.py.aui	2010-05-27 15:38:20.000000000 +0200
+++ wxPython-src-2.8.11.0/wxPython/wx/tools/Editra/src/ed_shelf.py	2010-05-27 15:38:28.000000000 +0200
@@ -28,7 +28,7 @@ import ed_glob
 from profiler import Profile_Get
 import plugin
 import iface
-import extern.aui as aui
+from wx.lib.agw import aui
 
 #--------------------------------------------------------------------------#
 # Globals
